### PR TITLE
Azure Kubernetes Services client credentials flow

### PR DESCRIPTION
**Pull request by noydavidi**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-10640)

## Description
Added Client Credentials Flow to integration.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@JudahSchwartz @michal-dagan Doc review completed.  Looks OK.

File: Packs/AzureKubernetesServices/Integrations/AzureKubernetesServices/AzureKubernetesServices_description.md [hunk 1]
```diff
@@ -49,3 +57,19 @@ Follow one of these steps for authentication based on Azure Managed Identities:
...
+To configure a Microsoft integration that uses this authorization flow with a self-deployed Azure application:
+   1. In the **Authentication Type** field, select the **Client Credentials** option.
+   2. In the **Application ID** field, enter your Client/Application ID.
+   3. In the **Tenant ID** field, enter your Tenant ID .
+   4. In the **Client Secret** field, enter your Client Secret.
+   5. Click **Test** to validate the URLs, token, and connection
+   6. Save the instance.
+
+### Testing authentication and connectivity
+If you are using Device Code Flow or Authorization Code Flow, for testing your authentication and connectivity to the Azure DevOps service run the ***!azure-devops-auth-test*** command.
```
Comment (michal-dagan) [inline Packs/AzureKubernetesServices/Integrations/AzureKubernetesServices/AzureKubernetesServices_description.md:74]: fix Azure DevOps references

File: Packs/AzureKubernetesServices/Integrations/AzureKubernetesServices/README.md [hunk 2]
```diff
@@ -41,6 +44,26 @@ Follow these steps for a self-deployed configuration:
...
+To configure a Microsoft integration that uses this authorization flow with a self-deployed Azure application:
+   1. In the **Authentication Type** field, select the **Client Credentials** option.
+   2. In the **Application ID** field, enter your Client/Application ID.
+   3. In the **Tenant ID** field, enter your Tenant ID .
+   4. In the **Client Secret** field, enter your Client Secret.
+   5. Click **Test** to validate the URLs, token, and connection
+   6. Save the instance.
+
+### Testing authentication and connectivity
+If you are using Device Code Flow or Authorization Code Flow, for testing your authentication and connectivity to the Azure DevOps service run the ***!azure-devops-auth-test*** command.
```
Comment (michal-dagan) [inline Packs/AzureKubernetesServices/Integrations/AzureKubernetesServices/README.md:64]: fix Azure DevOps references

**Review by michal-dagan — APPROVED** [review]
Great
just see my comments :)

**Review by JudahSchwartz — APPROVED** [review]

File: Packs/AzureKubernetesServices/Integrations/AzureKubernetesServices/AzureKubernetesServices_description.md [hunk 3]
```diff
@@ -49,3 +57,19 @@ Follow one of these steps for authentication based on Azure Managed Identities:
...
+To configure a Microsoft integration that uses this authorization flow with a self-deployed Azure application:
+   1. In the **Authentication Type** field, select the **Client Credentials** option.
+   2. In the **Application ID** field, enter your Client/Application ID.
+   3. In the **Tenant ID** field, enter your Tenant ID .
+   4. In the **Client Secret** field, enter your Client Secret.
+   5. Click **Test** to validate the URLs, token, and connection
+   6. Save the instance.
+
+### Testing authentication and connectivity
+If you are using Device Code Flow or Authorization Code Flow, for testing your authentication and connectivity to the Azure Kubernetes Services service run the ***!azure-devops-auth-test*** command.
```
Comment (michal-dagan) [inline Packs/AzureKubernetesServices/Integrations/AzureKubernetesServices/AzureKubernetesServices_description.md:74]:
```suggestion
If you are using Device Code Flow or Authorization Code Flow, for testing your authentication and connectivity to the Azure Kubernetes Services service run the ***!azure-ks-auth-test*** command. 
```

**Pull request merged**